### PR TITLE
fix(migrator): migrate environment document on identity migration

### LINKED ISSUE #991: Batch Edge migrator needs to copy all Environments to DynamoDB and set Projects to enable DynamoDB

### PR DESCRIPTION
**Pull request by gagantrivedi**

### CODE REVIEW

File: api/environments/dynamodb/dynamodb_wrapper.py [hunk 1]
```diff
@@ -5,24 +5,34 @@
...
+
+from environments.models import Environment
 
 if typing.TYPE_CHECKING:
     from environments.identities.models import Identity
 
 
-class DynamoIdentityWrapper:
+class DynamoWrapper:
+    TABLE_NAME: str = None
```
Comment (matthewelwell) [inline api/environments/dynamodb/dynamodb_wrapper.py]: I don't think this should be capitalised? By convention, only constants should be capitalised like this.

File: api/environments/dynamodb/dynamodb_wrapper.py [hunk 2]
```diff
@@ -5,24 +5,34 @@
...
+
     def __init__(self):
         self._table = None
-        if settings.IDENTITIES_TABLE_NAME_DYNAMO:
-            self._table = boto3.resource("dynamodb").Table(
-                settings.IDENTITIES_TABLE_NAME_DYNAMO
-            )
+        table = getattr(settings, self.TABLE_NAME)
```
Comment (matthewelwell) [inline api/environments/dynamodb/dynamodb_wrapper.py]: Based on this logic, I'm not sure that `table_name` is the correct attribute here, it's the name of the setting that holds the table name, right? So should it be `table_name_settings_attribute_name` perhaps? 

```python
table_name = getattr(settings, self.table_name_settings_attribute_name)
```

File: api/tests/unit/audit/test_unit_audit_signals.py [hunk 3]
```diff
@@ -1,87 +1,76 @@
-import json
-
-from flag_engine.api.document_builders import build_environment_document
+from pytest_django.asserts import assertQuerysetEqual
```
Comment (matthewelwell) [inline api/tests/unit/audit/test_unit_audit_signals.py]: I hate that this is camelcase 😂 

Is it bad that I'd kind of like us to do this:

```python
from pytest_django.asserts import assertQuerysetEqual as assert_queryset_equal
```

**Review by matthewelwell — COMMENTED** [review]
A few minor comments

File: api/environments/dynamodb/dynamodb_wrapper.py (hunk 1 above)
Comment (gagantrivedi) [reply api/environments/dynamodb/dynamodb_wrapper.py]: I think it's all the constants, even if they are class attributes? Although I personally don't like this either

Comment (matthewelwell) [reply api/environments/dynamodb/dynamodb_wrapper.py]: I'd say this isn't _really_ a constant though, right? For me, it's analogous to e.g. the `queryset` or the `permission_classes` attribute on a `ModelViewSet` class in DRF.

File: api/tests/unit/audit/test_unit_audit_signals.py (hunk 3 above)
Comment (gagantrivedi) [reply api/tests/unit/audit/test_unit_audit_signals.py]: Nope, I don't like camel case either: I can't understand why js like it

File: api/environments/dynamodb/dynamodb_wrapper.py (hunk 1 above)
Comment (gagantrivedi) [reply api/environments/dynamodb/dynamodb_wrapper.py]: Yeah, that makes sense - it felt odd

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**